runfolder/services.py

```diff
@@ -137,7 +163,8 @@ def get_runfolder_by_path(self, path):
 
         if not self._dir_exists(path):
             raise DirectoryDoesNotExist("Directory does not exist: '{0}'".format(path))
-        info = RunfolderInfo(self._host(), path, self.get_runfolder_state(path))
+        info = RunfolderInfo(self._host(), path, self.get_runfolder_state(path),
+                                self.get_metadata(path))
```

Comment:
This is nit-picking, but the indent of the level under should align so that the `self` statements align here.